stories/module-analytics-settings.stories.js

```diff
@@ -126,7 +125,6 @@ storiesOf( 'Analytics Module/Settings', module )
 		return (
 			<Settings
 				registry={ registry }
-				features={ [ 'ga4setup' ] }
```

Comment:
Some of these stories crash *but* they are already like this on develop https://google.github.io/site-kit-wp/storybook/develop/?path=/story/analytics-module-settings--no-tag-gtm-property-w-access

![already-failing](https://user-images.githubusercontent.com/8496063/127740003-0387816e-05ec-4951-915a-a75792b64112.png)